src/config/gnupg.md

```diff
@@ -0,0 +1,26 @@
+# GnuPG
+
+Void ships both GnuPG legacy (as `gnupg1`) and GnuPG stable (as `gnupg`).
+
+##  Smartcards
+
+For using smartcards such as Yubikeys with GnuPG, there are two backends for
+communicating with them through GnuPG: The internal CCID driver of GnuPG's
+scdaemon, or the PC/SC driver.
+
+## scdaemon with internal CCID driver
+
+By default, scdaemon, which is required for using smartcards with GnuPG, uses
+its internal CCID driver. For this to work, your smartcard needs to be one of
+the smartcards in the udev rules
+[here](https://salsa.debian.org/debian/gnupg2/-/blob/debian/unstable/debian/scdaemon.udev)
```

Comment:
let's link our version of these rules (https://github.com/void-linux/void-packages/blob/master/srcpkgs/gnupg/files/60-scdaemon.rules)

Reply:
Right, that was moved.

Reply:
ignore it :)